.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+## Overview
+**TL;DR**
+<One to two sentence description of the issue you are encountering.>
+
+### Extra Detail
+
+#### Steps to reproduce
+1.
+2.
+3.
+#### Screenshots
+<please upload images of the error to better showcase the problem.>
+
+#### Logs
+<please upload logs of the error to better showcase the problem.>
```

Comment:
Maybe could add something about using `--nodashboard`, `--logfile log.txt`, and `--loglevel trace`